addons/onboarding/models/onboarding_step.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class OnboardingStep(models.Model):
+    _name = 'onboarding.onboarding.step'
```

Comment:
Speaking of "_order", don't we need some kind of sequence field here to make steps display in a specific order?